src/odemis/gui/xmlh/resources/panel_tab_fastem_acqui.xrc

```diff
@@ -222,6 +222,7 @@
                     <border>10</border>
                   </object>
                 </object>
+                <bg>#4D4D4D</bg>
```

Comment:
I'm kind of surprised we hardcode styling like this. Commonly in UI programming sizes and colors are defined once, and then reused via variables. This greatly enhances consistency. There might be a variable system in xrc that could facilitate this. Obviously, not in scope for your work now.

Reply:
You are right! I quickly checked and it seems can one cannot define a constant in a XRC file. Maybe we can set an object's attribute e.g.
```
<attribute name="obj_bg">CONSTANT</attribute>
```

This is some refactoring work maybe for later. Can you approve this PR for now, if convinced @tmoerkerken ?

Reply:
Agreed it is completely out of scope for your work. Will approve. Might be an interesting topic for a Knowledge Sharing topic though 🤔